index.html

```diff
@@ -9,6 +9,32 @@
   <body class="page">
     <div class="page__container">
       <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <section class="contacts">
+        <h2 class="contacts__title">Остались вопросы?Свяжитесь с нами!</h2>
+        <ul class="contacts__list">
+          <li class="contacts__item">
+            <h3 class="contacts__heading" >Звоните:</h3>
+            <p class="contacts__access">+7(812) 555-66-66</p>
+          </li>
+          <li class="contacts__item contacts__item_type_adress">
+            <h3 class="contacts__heading" >Приезжайте в гости:</h3>
+            <p class="contacts__access">ул.Большая Конюшенная, 19/8</p>
+          </li>
+          <li class="contacts__item contacts__item_type_email">
+            <h3 class="contacts__heading" >Пишите:</h3>
+            <p class="contacts__access">mail@htmlacademy.ru</p>
```

Comment:
To make email functionality more direct and improve user experience, we can use the `<a>` tag with the `mailto` attribute instead of a paragraph with an email address.

```html
<a href="mailto:email@example.com">Send Email</a>
```

By using the `mailto` attribute in the `<a>` tag, emails become clickable and open a user's default email client, streamlining communication.

Reply:
@naumch1k I did it